build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/shared-configuration.kt

```diff
@@ -276,3 +280,181 @@ fun Project.resolver(name: String, libraryElements: String, extends: Configurati
         extendsFrom(extends)
     }
 }
+
+private val ignoreWithIsolatedProjectsExecuter = listOf(
```

Comment:
❌ please sort this list alphabetically (your IDE can do that for you)

Reply:
Good note, did it